front-end/src/components/settings-view/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './settings.module.scss';
+import PropTypes from 'prop-types';
+
+const Button = ({active, children, onClick}) => (
+    <a 
+        href='' 
```

Comment:
Maybe just use a button tag instead?

Reply:
I think you should allow for a tag `htmlTag` so that people can specify if they want button, a, or whatever